includes/class-sensei-quiz.php

```diff
@@ -1694,7 +1694,72 @@ public function get_questions( $quiz_id, $post_status = 'any', $orderby = 'meta_
 
 		$questions_query = new WP_Query( $question_query_args );
 
-		return $questions_query->posts;
+		$posts = $questions_query->posts;
+		return $filter_incomplete_questions ? $this->filter_out_incomplete_questions( $posts ) : $posts;
+	}
+
+	/**
+	 * Get all the questions of a quiz.
```

Comment:
Forgot to update the method description.

Reply:
Jup, done